proposals/stage-0/SECURITY_REPORTING/README.md

```diff
@@ -0,0 +1,60 @@
+# My New Proposal
+>  Stage 0
+
+## Champion
+
+Marcin Hoppe (@marcin_hoppe)
+
+## Description
+
+Currently OpenJS Foundation projects do not have consistently documented standards for responsible reporting and disclosure of security vulnerabilities. This proposal aligns OpenJS Foundation projects with industry best practices in this space.
+
+See also:
+- [Security reporting in OpenJS Foundation projects](https://gist.github.com/MarcinHoppe/b13a870770522c31a8386ada48b2e40f)
+- [Guidelines for responsible reporting and disclosure of security vulnerabilities](https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/security-guidelines.md)
+
+## Required Resources
+
+- Generic security policy template that projects can easily adopt
+- Optionally: vulnerability disclosure platform (VDP)
+
+## Who would be responsible?
+
+The responsibility for adoption of security reporting standards would be jointly owned by the projects and the CPC.
+
+## How would success be measured?
+
+- 100% of Impact, Growth, and At-Large projects have:
+    - Security policy on GitHub
+    - Vulnerability reporting mechanism (email or VDP)
+- 100% projects graduating from Incubation have a security policy on GitHub
+
+## Why this proposal is important
+
+Open source projects that can responsibly accept security reports and respond with a coordinated vulnerability disclosure have a higher chance of being treated as trustworthy by adopters.
+
+## Unresolved Question
+
+- Do we leave it to projects to coordinate disclosure?
+    - Some projects are already prepared, have infrastructure and processes in place. Others don't.
+- Do we need to offer additional resources to projects?
+    - Vulnerability disclosure platform (e.g. HackerOne)?
```

Comment:
perhaps best not to mention individual companies here? (nvm uses tidelift for this; snyk, whitesource, etc are also options)

Reply:
I'd like the platform to be one of the things can access to for free when joining the foundation (same way FOSS is offered). Don't know if it is possible to have one global account for the Foundation that allows granular access to members depending on the project (I definitely don't want to be aware of any high profile project vulnerability).
That could facilitate the triaging and making sure that nothing falls in the cracks (the foundation could receive all reports for the projects that opt-in and be prepared if there's no answer from the team). Also it could make it easier to handle bounties if in the future the foundation decides to offer them for every project (not to mix with corporate sponsors).

Reply:
@ljharb I only mentioned HackerOne because they are a member of the Foundation. I will remove this mention.

@molant we could have a single account where a triage team pulls maintainers into individual reports that only affect their projects. This is how the Node.js Ecosystem Security WG triage team operates today (https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md). The caveat is that it requires a triage team and is not completely self service for maintainers.

I can also get to know if or when GitHub plans to add security reporting mechanism to their platform.